utilities/lost-applymn.vbs

```diff
@@ -68,7 +69,7 @@ function search_for_info_in_note(date_of_app, applymn_confirmation_number, name_
 			case_note_header = trim(case_note_header)							'reformat the case note
 			If case_note_date <> "        " Then case_note_date = DateAdd("d", 0, case_note_date)		'make the date a date
 
-			If case_note_header ="~ Application Received (ApplyMN)" Then		'If it finds an application received case note for an ApplyMN
+			If case_note_header = "~ Application Received (ApplyMN)" or case_note_header = "~ Application Received (MNbenefi" Then		'If it finds an application received case note for an online application
```

Comment:
You have updated Application Received to use 'Mnbenefits' but this will look for 'MNbenefits' - it won't be able to find a match. This should probably have options to have a space and not and have it coded to a UCase so that any option works.

Reply:
Updates made. I also reached out to Faughn to see get HC information about the naming convention.